Makefile

```diff
@@ -49,18 +52,34 @@ build: docker_prepare
 # Docker
 docker_prepare:
 	-mkdir -p $(BUILD_DIR)/deb
-	docker-compose up --force-recreate -d
+	docker-compose $(DOCKER_COMPOSE_CONFIG) up --force-recreate -d
 ifeq ($(DEBUG),1)
 	-docker inspect $(CONTAINER_NAME)
 	-$(DOCKER_EXEC) "cat ~/.netrc"
 endif
 
 docker_image:
-	docker build -t $(IMAGE_NAME) ./docker
+	docker build -t $(PROJECT_NAME) ./docker
+
+docker_image_windows:
+	docker build -t $(PROJECT_NAME).windows -f docker/Dockerfile.windows .
 
 # GitHub
 github_upload:
-	$(DOCKER_EXEC) ". /usr/local/bin/tools.sh && create_release_and_upload_asset $(TRAVIS_TAG) $(DEB_NAME) $(DEB_MIME)"
+ifneq ($(BUILD_TAG),)
+ifeq ($(TRAVIS_OS_NAME),linux)
+	$(DOCKER_EXEC) "source docker/tools.sh && \
+	PROJECT_NAME=$(PROJECT_NAME) TRAVIS_OS_NAME=$(TRAVIS_OS_NAME) \
+	create_github_release $(BUILD_TAG)"
+else
+	brew install jq
+	source docker/tools.sh && \
+	osx_setup_fake_home /tmp && \
+	export PATH=$(PATH):$(shell pwd)/scripts/lib && \
+	HOME=/tmp PROJECT_NAME=$(PROJECT_NAME) TRAVIS_OS_NAME=$(TRAVIS_OS_NAME) create_github_release $(BUILD_TAG) && \
+	osx_clean_fake_home /tmp
```

Comment:
Same here for macos.
And the installation of tools deps needs to be done in `scripts/ci/osx/before_install.sh`

Reply:
Valid point.